packages/shared/src/components/modals/CompanionModal.tsx

```diff
@@ -41,110 +42,117 @@ export default function CompanionModal({
-          </Button>
+        <Button
+          className="mb-4 w-full btn-primary"
+          onClick={() => onActivateCompanion(url)}
+        >
+          Read and activate widget
+        </Button>
 
-          <div className="flex flex-col flex-1 justify-between">
-            <div className="hidden tablet:flex flex-col items-center">
-              <div className="flex items-center pb-4">
-                <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
-                  <LineIcon
-                    size={IconSize.Medium}
-                    secondary
-                    className="text-theme-color-cabbage"
-                  />
-                </div>
+        <Button
+          className="mb-6 w-full btn-tertiaryFloat"
+          onClick={onReadArticleClick}
+          tag="a"
+          target="_blank"
+          href={url}
+          rel="noopener"
+        >
+          Read post
+        </Button>
 
-                <span>Get a TLDR on the original post</span>
+        <div className="flex flex-col flex-1 justify-between">
+          <div className="hidden tablet:flex flex-col items-center">
+            <div className="flex items-center pb-4">
+              <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
+                <UnreadIcon
+                  size={IconSize.Medium}
+                  secondary
+                  className="text-theme-color-cabbage"
+                />
               </div>
-              <div className="hidden tablet:flex items-center pb-4">
-                <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
-                  <CommentIcon
-                    size={IconSize.Medium}
-                    secondary
-                    className="text-theme-color-blueCheese"
-                  />
-                </div>
-                <span>Discuss from everywhere</span>
+
+              <span>Get a TLDR on the original post</span>
+            </div>
+            <div className="hidden tablet:flex items-center pb-4">
+              <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
+                <CommentIcon
+                  size={IconSize.Medium}
+                  secondary
+                  className="text-theme-color-blueCheese"
+                />
               </div>
-              <div className="hidden tablet:flex items-center pb-4">
-                <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
-                  <UpvoteIcon
-                    secondary
-                    size={IconSize.Medium}
-                    className="text-theme-color-avocado"
-                  />
-                </div>
-                <span>Bookmark with a single click</span>
+              <span>Discuss from everywhere</span>
+            </div>
+            <div className="hidden tablet:flex items-center pb-4">
+              <div className="inline-block p-2 mr-2 bg-theme-active rounded-16">
```

Comment:
These should have the respective color as background.
For avocado for instance: `bg-theme-overlay-float-avocado`

Reply:
@rebelchris looks like `bg-theme-overlay-blueCheese` & `-bun` are not defined, I tried adding these, but I think I am missing something as it does not seem to register the new color. What are all the places which need to be updated to add a new color?

Reply:
Hmm those should be ok, not sure if the float was needed.
I'll check now :)